packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.js

```diff
@@ -96,6 +97,13 @@ export const BrowseStep = ({
     }
   );
 
+  const singularTypes = toSingularTypes(allowedTypes);
+  const assets = rawAssets.map((asset) => {
+    const fileType = asset?.mime?.split('/')?.[0];
```

Comment:
do you think it would be more readable passing `asset?.mime` to `isSelectable` with default to empty string inside? 👀 @joshuaellis 
will also add type doc inside the util before merging

Reply:
Yeah that's not a bad idea 👍🏼 less ternaries